OloEngine/tests/Networking/NetworkIntegrationTest.cpp

```
@@ -0,0 +1,295 @@
+
+    NetworkManager::StopServer();
+}
+
+TEST_F(NetworkIntegrationTest, ServerTracksConnectedClient)
+{
+    ASSERT_TRUE(NetworkManager::StartServer(kTestPort));
+    ASSERT_TRUE(NetworkManager::Connect("127.0.0.1", kTestPort));
+    ASSERT_TRUE(WaitUntil([]
+                          { return NetworkManager::IsConnected(); }));
+
+    auto const* server = NetworkManager::GetServer();
```

Clang (14.0.6)

[warning] variable 'server' is not initialized

(cppcoreguidelines-init-variables)